runtime/src/lib.rs

```diff
@@ -287,7 +287,7 @@ impl pallet_grandpa::Config for Runtime {
 		pallet_grandpa::EquivocationReportSystem<Self, Offences, Historical, ReportLongevity>;
 
 	type WeightInfo = ();
-	type MaxAuthorities = ConstU32<32>; // Q: maximum number of grandpa authorities?
+	type MaxAuthorities = ConstU32<256>;
```

Comment:
How does this relate to the number of active validators? Do all active validators participate in the finalization? (and therefore, does this number need to always exceed active validators?)

Reply:
Spoke with Nathan about this last week. The warning that is being issued that caused the ticket to be created came exclusively from `pallet_grandpa`. So it is only the authorities for grandpa that are incorrect. They are the one item in substrate that come with very little documentation. Nathan suggested we need at least as many authorities as we have validators which he said is 200. Bumped it up to the nearest power of two to give us some buffer room. The test code for substrate had it at 100 so this answer seems reasonably correct based on that. However, the code that is actually throwing the warning is a weakly bounded vector. So when you try to pass in more items than it accepts (max items defined by the MaxAuthorities) it can throw an optional warning message without actually breaking (which is what happened). So 256 is a good guess for right now based on what Nathan said but the risk of being "wrong" is practically zero